src/main/java/net/sf/jabref/gui/FileHistoryMenu.java

```diff
@@ -88,16 +69,13 @@ public void newFile(String filename) {
 
     private void setItems() {
         removeAll();
-        Iterator<String> i = history.iterator();
-        int count = 1;
-        while (i.hasNext()) {
-            addItem(i.next(), count);
-            count++;
+        for (int count = 0; count < history.size(); count++) {
+            addItem(history.getFileName(count), count + 1);
         }
     }
 
     private void addItem(String filename, int num) {
-        String number = num + "";
+        String number = Integer.toString(num);
```

Comment:
String.valueOf = 60 hits, Integer.toString = 32 hits. So I kept it, but can make a general search-and-replace if that is deemed to be the correct solution.